internal/util/projutil/project_util.go

```diff
@@ -26,10 +26,7 @@ import (
 )
 
 const (
-	// Useful file modes.
```

Comment:
These should be kept around and used throughout the project.

Reply:
They are not used at all. It is dead code.

Reply:
They should/will be. No exported code is dead code btw.

Reply:
If one day we need any other permission them we add. But now, they are not used at all. I mean, it is dead code that just passes in the lint because is exported.

Reply:
oks.